inlong-agent/agent-installer/src/main/java/org/apache/inlong/agent/installer/ModuleManager.java

```diff
@@ -461,7 +462,7 @@ private boolean downloadModule(ModuleConfig module) {
     private boolean isPackageDownloaded(ModuleConfig module) {
         String path = module.getPackageConfig().getStoragePath() + "/" + module.getPackageConfig().getFileName();
         String fileMd5 = calcFileMd5(path);
-        if (fileMd5.equals(module.getPackageConfig().getMd5())) {
+        if (Objects.equals(fileMd5, module.getPackageConfig().getMd5())) {
```

Comment:
what if  both`fileMd5` and `module.getPackageConfig().getMd5()` are null.

Reply:
After obtaining the module configuration, it is necessary to add a check, md5 cannot be null

Reply:
If this situation occurs currently, there will be repeated pulling of installation packages